src/test/java/api/support/APITests.java

```diff
@@ -619,4 +629,45 @@ protected void mockClockManagerToReturnFixedDateTime(DateTime dateTime) {
   protected void mockClockManagerToReturnDefaultDateTime() {
     ClockManager.getClockManager().setDefaultClock();
   }
+
+  public class CirculationPolicies {
```

Comment:
The test uses fixtures in order to pre-define some default configuration. It can not be extracted.

Reply:
I don't understand, why does the use of fixtures define where this class can be defined?

Reply:
The fixtures are members of APITests class, and `CirculationPolicies` class uses them directly from outer class. If this class is defined separately, then we will have to inject those dependencies every time we create the instance of class, which reduces usability of this approach.

Reply:
Ok, I didn't realise this class used fields from the outer scope.

This class is really big. The proliferation of fixture fields is a challenge in this area. At some point I'd like to reduce that. Changing this might come at a similar time.